shark/examples/shark_training/bert_training_load_tf.py

```diff
@@ -27,14 +27,10 @@
         np.random.randint(5, size=(BATCH_SIZE, SEQUENCE_LENGTH)),
         np.random.randint(5, size=(BATCH_SIZE, SEQUENCE_LENGTH))
     ]
-    file_link = "https://storage.googleapis.com/shark_tank/users/stanley/bert_tf_training.mlir"
-    response = request.urlretrieve(file_link, load_args.download_mlir_path)
+    model_name = "bert_tf_training"
```

Comment:
I think the model name should be equivalent to the hugging face model name with {training} and {inference} at the end. Users will find it easy to run the model.

Reply:
agreed. I would even drop _inference at the end and assume inference by default. Beside for training it could just be a flag to shark_load() to get the training graph and what framework etc.

Reply:
this is not a huggingface model per se, but we do need a nice way to name it :)